include/deal.II/lac/block_matrix_base.h

```diff
@@ -377,12 +377,15 @@ class BlockMatrixBase : public Subscriptor
   ~BlockMatrixBase ();
 
   /**
-   * Copy the given matrix to this one.  The operation throws an error if the
-   * sparsity patterns of the two involved matrices do not point to the same
-   * object, since in this case the copy operation is cheaper. Since this
-   * operation is nonetheless not for free, we do not make it available through
-   * operator=(), since this may lead to unwanted usage, e.g. in copy
-   * arguments to functions, which should really be arguments by reference.
+   * Copy the matrix given as argument into the current object.
+   *
+   * Since copying matrices is an expensive operation that we do not want to
+   * happen by accident through compiler generated code to <code>operator=</code>
+   * (as would happen, for example, if one accidentally declared a function
+   * argument of the current type <i>by value</i> rather than <i>by reference</i>),
+   * the functionality of copying matrices is implemented in this member
+   * function instead. All copy operations of objects of this type therefore
```

Comment:
@bangerth This is a monster of a sentence spanning over 5 lines. Although the German part in me feels some satisfaction in reading through it, I'd suggest to break it up.
```
Copying matrices is an expensive operation that we do not want to happen
by accident through compiler generated code for <code>operator=</code>.
(This would happen, for example, if one accidentally declared a function
argument of the current type <i>by value</i> rather than
<i>by reference</i>.) The functionality of copying matrices is implemented
in this member function instead. [...]
```